code/src/java/pcgen/gui2/dialog/DataInstaller.java

```diff
@@ -439,15 +440,9 @@ public DataInstaller()
 	 */
 	private boolean checkNonStandardOK(Collection<String> files)
 	{
-		Collection<String> nonStandardFiles = new ArrayList<>();
-		for (String filename : files)
-		{
-			if (!filename.toLowerCase().startsWith(DATA_FOLDER)
-				&& !filename.toLowerCase().startsWith(OUTPUTSHEETS_FOLDER))
-			{
-				nonStandardFiles.add(filename);
-			}
-		}
+		Collection<String> nonStandardFiles =
+				files.stream().filter(filename -> !filename.toLowerCase().startsWith(DATA_FOLDER)
```

Comment:
Sorry, this fails the readability tests for me as a stream.  The embedded check is too big - extract or somehow make this simpler.